### DISCUSSION TITLE
Fix Quark colored flower pot crashes

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#182] [Bugfix Suggestions]
https://mclo.gs/5bfy8mf
Been an issue for years now that's especially affecting dedicated servers for some reason.

**Comment by ACGaming** [comment]
Closed in favor of Quark fork: https://github.com/ACGaming/Quark/commit/a07c8f74ea9ff6014baefb414b65b092dad46ee1
